op-chain-ops/deployer/state/state.go

```diff
@@ -75,7 +75,6 @@ type ChainState struct {
 	L1CrossDomainMessengerProxyAddress        common.Address `json:"l1CrossDomainMessengerProxyAddress"`
 	OptimismPortalProxyAddress                common.Address `json:"optimismPortalProxyAddress"`
 	DisputeGameFactoryProxyAddress            common.Address `json:"disputeGameFactoryProxyAddress"`
-	DisputeGameFactoryImplAddress             common.Address `json:"disputeGameFactoryImplAddress"`
```

Comment:
Note that another `DisputeGameFactoryImplAddress` appears on Line 62 above. This is a duplicate, and because the DGF supports MCP, we do not want to deploy a unique DGF impl per chain.